tuf/exceptions.py

```diff
@@ -173,7 +173,7 @@ def __init__(self, expected_length, observed_length):
 
   def __str__(self):
     return 'Observed length (' + repr(self.observed_length)+\
-           ') <= expected length (' + repr(self.expected_length) + ').'
+           ') != expected length (' + repr(self.expected_length) + ').'
```

Comment:
The reason why I changed it` from `<=` to `!= is because if there is a bug in the way the download function works, nothing actually prevents it from downloading more than it should. This happened to me, and I found the error message very misleading (e.g., `9 <= 5`).

Reply:
Behavior here has not changed. It should not be possible to get a length greater than expected due to the way the download code was written (and now as well, I think). If the code is changed and it becomes possible to end up with more, we can change the error message, but '<' currently communicates an expectation that has some value.